django2pydantic/handlers/relational.py

```
@@ -1,114 +1,112 @@
         return None
 
+    @override
+    def get_pydantic_type_raw(
+        self,
+    ) -> UnionType | SupportedPydanticTypes | list[SupportedPydanticTypes]:
+        """Return the Pydantic type of the field."""
+        if not isinstance(self.field_obj, RelatedField):
+            raise TypeError(
+                f"Expected 'field_obj' to be of type 'RelatedField', "
+                f"got {type(self.field_obj)} instead"
+            )
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)